glide.lock

```diff
@@ -112,21 +112,21 @@ imports:
   - leveldb/table
   - leveldb/util
 - name: golang.org/x/crypto
-  version: f027049dab0ad238e394a753dba2d14753473a04
+  version: de0752318171da717af4ce24d0a2e8626afaeb11
```

Comment:
Why do they change all the time?

Reply:
Because of glide update. If you add new dep - glide update must be run. If some deps uses master version - they are upgraded to latest commit at the moment